sgx-tools/dcap_linux.go

```diff
@@ -0,0 +1,506 @@
+package main // import "github.com/inclavare-containers/sgx-tools"
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-restruct/restruct"
+	"github.com/golang/protobuf/proto"
+	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	pb "github.com/inclavare-containers/sgx-tools/proto"
+	"github.com/sirupsen/logrus"
+	"net"
+	"strings"
+)
+
+const (
+	aesmdSocket                  = "/var/run/aesmd/aesm.socket"
+	AttestationKeyIdentityLength = 256
+	nonceLength                  = 16
+	qeReportInfoLength           = 960
+	// In millisecond
+	timeOut          = 15000
+	rawMessageLength = 4
+)
+
+// dialAesmd: a duplicated function inherited from intelsgx package
+func dialAesmd() (*net.UnixConn, error) {
```

Comment:
Plz remove the unnecessary space.

Reply:
> Plz remove the unnecessary space.

Do you mean to change to `func dialAesmd()(*net.UnixConn, error) {`  or `func dialAesmd() (*net.UnixConn, error){` ?  However, the codes will be automatically generated by `go fmt` as `func dialAesmd() (*net.UnixConn, error) {`

Reply:
Just ignore this. I'm blinded.